parsers/to-react-native/README.md

```diff
@@ -0,0 +1,150 @@
+# To React Native
+
+## Description
+
+Transform design tokens to a JavaScript `theme` object which can be used in React Native.
+
+## Usage
+
+### Options
+
+| Parameter          | Required | Type                                                                       | Default     | Description                                                                                                           |
+| ------------------ | -------- | -------------------------------------------------------------------------- | ----------- | --------------------------------------------------------------------------------------------------------------------- |
+| `colorFormat`      | optional | `rgb`, `prgb`, `hex`, `hex6`, `hex3`, `hex4`, `hex8`, `name`, `hsl`, `hsv` | `rgb`       | The format of all colors (gradients, borders, textStyles etc.)                                                        |
+| `objectName`       | optional | `string`                                                                   | `'theme'`   | The name of the JS object containing the theme,                                                                       |
+| `assetsFolderPath` | optional | `string`                                                                   | `undefined` | The relative location of the folder to import the assets from, if not provided, the assets will be referenced by URL. |
+| `prettierConfig`   | optional | [`prettier.Config`](https://prettier.io/docs/en/options.html)              | `undefined` | Formatting options of the file.                                                                                       |
```

Comment:
Please list all the Prettier config properties this parser supports. You can find an example of how we display this in the `options` section here: https://github.com/Specifyapp/parsers/blob/master/parsers/to-scss-mixin-text-style/README.md#interface

Thanks 🙏